tools/fuchsia/copy_debug_symbols.py

```diff
@@ -26,10 +26,8 @@ def Touch(fname):
 
 
 def GetBuildIdParts(exec_path, read_elf):
-  file_out = subprocess.check_output(
-      [read_elf, '--hex-dump=.note.gnu.build-id', exec_path])
-  second_line = file_out.splitlines()[-1].split()
-  build_id = second_line[1] + second_line[2]
+  file_out = subprocess.check_output([read_elf, '-n', exec_path])
+  build_id = file_out.splitlines()[-1].split()[-1]
```

Comment:
I'd also vote for more error message here. Something like, "Tried to find a hex string of the SHA1 hash of the text sections in the ELF file as the build_id. But failed because of...".

Reply:
Yeah that would be better.

Is it safe to assume it will be a SHA-1 hex string?  That way we can assert on the length and content.

Reply:
If we change it, I'd rather it fail right away. Also, please log that we are explicitly looking for a SHA1 hex string in the build-id when you assert. Asserting here would definitely be better than failing because we failed to create some directory later. This error manifested way to far away from where the issue was occurred.

Reply:
Updated - PTAL